src/lib/select/select.ts

```diff
@@ -101,9 +101,6 @@ let nextUniqueId = 0;
  * the trigger element.
  */
 
-/** The max height of the select's overlay panel */
-export const SELECT_PANEL_MAX_HEIGHT = 256;
```

Comment:
The CI is failing for this, because it's exported which means that removing is technically a breaking change.